script/hassfest/translations.py

```diff
@@ -42,6 +42,7 @@
     "rpi_power",
     "waze_travel_time",
     "zodiac",
+    "remootio",
```

Comment:
Remootio is a brand thus should not be in this list

Reply:
`remootio` is also the name of the integration. I don't know how this file will be generated but I mean based on the names of the integrations.

Reply:
The point of this map is to allow translating integration names which are not brand names, and the reason you had to add remootio here is because you've added a translated title to `strings.json`. Since Remootio is a brand name, please remove it from here and from strings.json.

Reply:
OK, I will do.